src/dlgAboutDialog.cpp

```diff
@@ -94,4 +98,696 @@ dlgAboutDialog::dlgAboutDialog(QWidget* parent) : QDialog(parent)
+                   "<p><span style=\"color:#bc8942;\"><big><b>Damian Monogue</b></big></span> (<span style=\"color:#40b040;\">demonnic</span> <span style=\"color:#0000ff;\">demonnic@gmail.com</span>) former maintainer of the early Windows and Apple OSX packages. He also administers our server and helps the project in many ways.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><big><b>Florian Scheel</b></big></span> (<span style=\"color:#40b040;\">keneanung</span> <span style=\"color:#0000ff;\">keneanung@googlemail.com</span>) contributed many improvements to Mudlet's db: interface, event system, and has been around the project for a very long while assisting users.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><big><b>Ahmed Charles</b></big></span> (<span style=\"color:#40b040;\">ahmedcharles</span> <span style=\"color:#0000ff;\">acharles@outlook.com</span>) contributions to the Travis integration, CMake and Visual C++ build, a lot of code quality and memory management improvements.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><big><b>Chris Mitchell</b></big></span> (<span style=\"color:#40b040;\">Chris7</span> <span style=\"color:#0000ff;\">chrismudlet@gmail.com</span>) has developed a shared module system that allows script packages to be shared among profiles, a UI for viewing Lua variables, improvements in the mapper and all around.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Ben Carlsen</b></span> (<span style=\"color:#0000ff;\">arkholt@gmail.com</span>) has developed the first version of our Mac OSX installer. He is the former maintainer of the Mac version of Mudlet.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Ben Smith</b></span> () joined in December 2009 though he's been around much longer. Contributed to the Lua API and is the former maintainer of the Lua API.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Blaine von Roeder</b></span> () joined in December 2009. He has contributed to the Lua API, submitted small bugfix patches and has helped with release management of 1.0.5.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Bruno Bigras</b></span> (<span style=\"color:#0000ff;\">bruno@burnbox.net</span>) developed the original cmake build script and he has committed a number of patches.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Carter Dewey</b></span> (<span style=\"color:#0000ff;\">eldarerathis@gmail.com</span>) contributions to the Lua API.</p>\n"
+                   "<p><span style=\"color:#bc8942;\"><b>Erik Pettis</b></span> (<span style=\"color:#40b040;\">Oneymus</span>) developed the Vzyor GUI Manager for Mudlet.</p>\n"
```

Comment:
That would be _Vyzor_. He also helps users in the Discord channel!

Reply:
:blush: um, yeah _Vyzor_...

Reply:
Fixed.